Sources/Workspace/InitPackage.swift

```diff
@@ -290,7 +290,12 @@ public final class InitPackage {
           : sources.appending(component: "\(pkgname)")
         try makeDirectories(moduleDir)
 
-        let sourceFileName = "\(typeName).swift"
+        let sourceFileName: String
+        if packageType == .executable || packageType == .tool {
```

Comment:
`tool` uses an `@main` struct for its entry point. Will using `main.swift` continue to work with that?

Reply:
Oh, I missed that, we probably have to exempt `.tool` from this change then, but I'll double-check

Reply:
LG now 👍